conf/modules.config

```diff
@@ -0,0 +1,5 @@
+process {
```

Comment:
nf-core modules have to be configured through the `modules.config`. It is a bit clunky but the positive is that the workflow logic and configuration are separate.